_config/pom.yml

```diff
@@ -14,55 +14,55 @@ kieExecutionServerDescription: Standalone execution server that can be used to r
   Distribution zip contains WAR files for all supported containers.
 
 latestFinal:
-    version: 7.10.0.Final
-    releaseDate: 2018-08-17
+    version: 7.11.0.Final
+    releaseDate: 2018-09-10
 
-    droolsZip: https://download.jboss.org/drools/release/7.10.0.Final/drools-distribution-7.10.0.Final.zip
+    droolsZip: https://download.jboss.org/drools/release/7.11.0.Final/drools-distribution-7.11.0.Final.zip
 
-    droolsjbpmIntegrationZip: https://download.jboss.org/drools/release/7.10.0.Final/droolsjbpm-integration-distribution-7.10.0.Final.zip
+    droolsjbpmIntegrationZip: https://download.jboss.org/drools/release/7.11.0.Final/droolsjbpm-integration-distribution-7.11.0.Final.zip
 
-    droolsWorkbenchEAP7WAR: https://download.jboss.org/drools/release/7.10.0.Final/kie-drools-wb-7.10.0.Final-eap7.war
-    droolsWorkbenchWildFlyWAR: https://download.jboss.org/drools/release/7.10.0.Final/kie-drools-wb-7.10.0.Final-wildfly11.war
-    droolsWorkbenchTomcatWAR: https://download.jboss.org/drools/release/7.10.0.Final/kie-drools-wb-7.10.0.Final-tomcat8.war
+    droolsWorkbenchEAP7WAR: https://download.jboss.org/drools/release/7.11.0.Final/kie-drools-wb-7.11.0.Final-eap7.war
+    droolsWorkbenchWildFlyWAR: https://download.jboss.org/drools/release/7.11.0.Final/kie-drools-wb-7.11.0.Final-wildfly11.war
+    droolsWorkbenchTomcatWAR: https://download.jboss.org/drools/release/7.11.0.Final/kie-drools-wb-7.11.0.Final-tomcat8.war
 
-    droolsjbpmToolsZip: https://download.jboss.org/drools/release/7.10.0.Final/droolsjbpm-tools-distribution-7.10.0.Final.zip
+    droolsjbpmToolsZip: https://download.jboss.org/drools/release/7.11.0.Final/droolsjbpm-tools-distribution-7.11.0.Final.zip
 
-    documentationHtmlSingle: https://docs.jboss.org/drools/release/7.10.0.Final/drools-docs/html_single/index.html
+    documentationHtmlSingle: https://docs.jboss.org/drools/release/7.11.0.Final/drools-docs/html_single/index.html
 
-    KIE_API_documentationJavadoc: https://docs.jboss.org/drools/release/7.10.0.Final/kie-api-javadoc/index.html
+    KIE_API_documentationJavadoc: https://docs.jboss.org/drools/release/7.11.0.Final/kie-api-javadoc/index.html
 
-    droolsWhatsNew: https://docs.jboss.org/drools/release/7.10.0.Final/drools-docs/html_single/#_droolsreleasenoteschapter
+    droolsWhatsNew: https://docs.jboss.org/drools/release/7.11.0.Final/drools-docs/html_single/#_droolsreleasenoteschapter
     droolsReleaseNotes: https://issues.jboss.org/secure/ReleaseNote.jspa?projectId=12313021&version=12338781
```

Comment:
We should update to: https://issues.jboss.org/secure/ReleaseNote.jspa?projectId=12313021&version=12338870